### PR TITLE
append -v1 to image name

### PR DESCRIPTION
**Pull request by mphillips-infoblox**
Updating image name to match `-v1` name suffix used for ci/cd pipeline.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Docker hub repo
 REGISTRY ?= infoblox
 # image name
-IMAGE_NAME ?= db-controller
+IMAGE_NAME ?= db-controller-v1
```
Comment (kd7lxl) [inline Makefile:4]: I get an error now:
```
% make build-chart
Error: stat helm/db-controller-v1: no such file or directory
```

**Review by kd7lxl — CHANGES_REQUESTED** [review]
Can you clarify what problem this is solving? It seems really odd to me to put the version number in the name, then prompting append another version number to the end of that (e.g., `db-controller-v1-v0.0.1-32-g1f7123e.tgz`).

File: helm/db-controller-legacy-crds/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: db-controller-crds
+name: db-controller-crds-legacy
```
Comment (kd7lxl) [inline helm/db-controller-legacy-crds/Chart.yaml:2]:
```suggestion
name: db-controller-legacy-crds
```

File: helm/db-controller-legacy/values.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Default values for db-controller.
+# Default values for db-controller-v1.
```
Comment (kd7lxl) [inline helm/db-controller-legacy/values.yaml:1]: Don't need `-v1`

File: build.properties
[the remark this answers is not in the document]
Comment (mphillips-infoblox) [reply build.properties:1]: done

File: helm/db-controller-legacy-crds/Chart.yaml (hunk 2 above)
Comment (mphillips-infoblox) [reply helm/db-controller-legacy-crds/Chart.yaml]: done.

File: helm/db-controller-legacy/values.yaml (hunk 3 above)
Comment (mphillips-infoblox) [reply helm/db-controller-legacy/values.yaml]: done.

**Review by kd7lxl — APPROVED** [review]

**Pull request merged**